src/main/java/com/esri/samples/editing/update_geometries/UpdateGeometriesSample.java

```diff
@@ -108,8 +109,11 @@ public void start(Stage stage) throws Exception {
                 featureLayer.clearSelection();
                 featureLayer.selectFeature(selected);
               } else {
-                // move selected features
-                moveSelected(mapPoint);
+                FeatureQueryResult result = featureLayer.getSelectedFeaturesAsync().get();
```

Comment:
Can you remove changes for this sample. I have a fix that handles a couple other things as well in another PR: https://github.com/Esri/arcgis-runtime-samples-java/pull/176.